# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xmms-winamp-x-theme/colors.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some slithgly tuned, visible selection color.

  - Rene Rebe <rene@exactcode.de>

diff -u Winamp_X_XMMS_1.01-orig/pledit.txt Winamp_X_XMMS_1.01/pledit.txt
--- Winamp_X_XMMS_1.01-orig/pledit.txt	2001-05-22 15:13:46.000000000 +0200
+++ Winamp_X_XMMS_1.01/pledit.txt	2005-08-05 19:41:19.000000000 +0200
@@ -5,7 +5,7 @@
 NormalBG=#9AC1EA
 Normal=#000000
 Current=#3966A0
-SelectedBG=#9AC1EA
+SelectedBG=#BAE1FF
 Font=Tahoma
 mbBG=#FFFFFF
-mbFG=#000000
\ No newline at end of file
+mbFG=#000000
